### DISCUSSION TITLE
Change operator log verbosity

### DISCUSSION THREAD

**Discussion by pbarois** [kube-logging/logging-operator#1669] [Q&A] [answered]
Hello,

i can't find how to change the operator log verbosity from "info" to "error".
Can you point me how i can do that ?

Thanks.

Pierre.

**Comment by pepov** [comment]
does this help? https://kube-logging.dev/docs/operation/troubleshooting/fluentd/#set-fluentd-log-level

**Reply by pbarois** [reply]
thanks for the quick answer, but i want to change the log level of "logging-operator" image.
Eg: https://github.com/kube-logging/logging-operator/blob/master/main.go

I have already change the level for fleuntd with the link you have send.

**Comment by pbarois** [comment]
after reading https://sdk.operatorframework.io/docs/building-operators/golang/references/logging/, look like we can't change the log level for the moment. Only log format and "development" mode can be updated.

**Comment by pepov** [comment, marked as answer]
We don't use the operator framework actually, but what you found applies to our case and you can raise the log verbosity, not decrease. If you are looking to have that feature in the future you are welcome to open an issue for that.
